usecase/usecase-webapi-xwalk-tests/samples/GetUserMedia/LICENSE.md

```diff
@@ -0,0 +1,29 @@
+Copyright (c) 2014, The WebRTC project authors. All rights reserved.
```

Comment:
It's from https://github.com/webrtc/samples/tree/gh-pages/src/content/getusermedia/gum